pkg/server/plugin/upstreamauthority/awspca/pca_test.go

```diff
@@ -188,6 +190,45 @@ func (as *PCAPluginSuite) Test_MintX509CA() {
 	as.Require().Equal([][]byte{expectedRoot.Raw}, response.UpstreamX509Roots)
 }
 
+func (as *PCAPluginSuite) Test_MintX509CA_WithSupplementalBundle() {
```

Comment:
Looks like only a happy path is tested here. Maybe we can test loading a malformed supplemental bundle?

Reply:
I had originally excluded this test because it is exercising error detection logic in a helper function (same as dedupe logic). I added a case for it though in the configure part of the suite. Would you also prefer an assertion on the dedupe functionality?

Reply:
I see. Asserting the dedupe functionality would be nice to have, but I'm ok with that not being part of this PR.

Reply:
Hey @amartinezfayo, I ended up modifying the test fixture to exercise the dedupe functionality as well. After some more thought, it did feel important to preserve. Mind taking a quick look?